doc/user_guide/datamodels.md

```diff
@@ -0,0 +1,65 @@
+Representing a datamodel (entity)
+----------------------------------
+
+The (concepts)[concepts.md] describes the underlying structure of DLite datamodels.
```

Comment:
`[concepts](concepts.md)` will create a link to `concepts.md`, but this file does not exists in the folder of the generated html documentation. Better to add a static link by writing `[concepts]` here and add the link

    [concepts]: https://sintef.github.io/dlite/user_guide/concepts.html

to the end of the document.




```suggestion
The [concepts] describes the underlying structure of DLite datamodels.
```